doc/2/api/controllers/document/m-replace/index.md

```diff
@@ -83,12 +83,12 @@ kourou document:mReplace <index> <collection> <body> -a silent=true
 
 - `collection`: collection name
 - `index`: index name
-- `silent`: if set, then Kuzzle will not generate notifications <SinceBadge version="2.9.2" />
 
 ### Optional:
 
 - `refresh`: if set to `wait_for`, Kuzzle will not respond until the replacements are indexed
 - `strict`: if set, an error will occur if at least one document has not been replaced <SinceBadge version="auto-version" />
+- `silent`: if set, then Kuzzle will not generate notifications <SinceBadge version="2.9.2" />
```

Comment:
Hmmm this is a tricky one, We can't just move `silent` option to **optional** and say it was here since **2.9.2**.
Silent option exists since `2.9.2`? But now it is optional since `auto-version`?
@scottinet

Reply:
It was always optional and it exists since 2.9.2